### PR TITLE
test(etfw): logging improvements

### PR DESCRIPTION
**Pull request by chriswldenyer**
Allow a user-specified name to be passed to the deploy script to be returned in event and test run reports.
Modify the test pipeline to include the build number in the name.
Renamed some variables to avoid ambiguity. Slight reordering of some code.
Custom resource monitoring updates every 4 minutes to avoid the command timing out when using kubectl log --follow
Fixed cases where failure events were sent multiple times.

### CODE REVIEW

File: src/tools/extended-test-framework/test_conductor/tests/test_common.go [hunk 1]
```diff
@@ -192,9 +192,10 @@ func MonitorCRs(
 
 	var endTime = time.Now().Add(duration)
 	var waitSecs = 5
-	for i := 0; ; i++ {
-		if i%100 == 0 {
-			logf.Log.Info("Monitoring CRs", "seconds elapsed", i*waitSecs)
+	var progressSecs = 240
+	for elapsedSecs := 0; ; elapsedSecs += waitSecs {
```
Comment (milanhajek) [inline src/tools/extended-test-framework/test_conductor/tests/test_common.go:196]: If I understand the code correctly, I'd prefer `elapsedSecs = 240` and then after timeout do elapsedSecs -= waitSecs. `for elapsedSecs <= 0 {`

Comment (chriswldenyer) [reply src/tools/extended-test-framework/test_conductor/tests/test_common.go:196]: elapsedSecs isn't meant to control the number of iterations in the loop, it it just used to log how long the loop has been running and just goes up indefinitely. The loop gets terminated with:
    if time.Now().After(endTime) { ...
Ok, maybe the way I used the for loop is unconventional... 
I suppose I could use just "for {" and put the other stuff inside the loop,

Comment (chriswldenyer) [reply src/tools/extended-test-framework/test_conductor/tests/test_common.go]: Changed loop structure as per my comment.

**Review by milanhajek — APPROVED** [review]

File: src/tools/extended-test-framework/test_conductor/tests/test_common.go (hunk 1 above)
Comment (milanhajek) [reply src/tools/extended-test-framework/test_conductor/tests/test_common.go]: Thanks

**Review by rohan2794 — APPROVED** [review]
LGTM

**Pull request merged**